src/vstart.sh

```diff
@@ -899,12 +911,15 @@ EOF
             if command -v btrfs > /dev/null; then
                 for f in $CEPH_DEV_DIR/osd$osd/*; do btrfs sub delete $f &> /dev/null || true; done
             fi
+            mkdir -p $CEPH_DEV_DIR/osd$osd
```

Comment:
This would break the filestore_path and kstore_path portions below.

Reply:
@athanatos updated.